`text2image` links to pango_training, but upstream's CMake configuration
doesn't cause pango_training to actually get installed, making `text2image`
unusable.
---
diff --git a/src/training/CMakeLists.txt b/src/training/CMakeLists.txt
index c764442..cf7f062 100644
--- a/src/training/CMakeLists.txt
+++ b/src/training/CMakeLists.txt
@@ -439,6 +439,11 @@ if(ICU_FOUND)
     endif()
     target_include_directories(pango_training
                                PUBLIC pango ${CMAKE_CURRENT_BINARY_DIR})
+    install(
+      TARGETS pango_training
+      RUNTIME DESTINATION bin
+      LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+      ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
     generate_export_header(pango_training EXPORT_MACRO_NAME
                            TESS_PANGO_TRAINING_API)
     project_group(pango_training "Training Tools")
